analyses/doublet-detection/scripts/01a_run-scdblfinder.R

```diff
@@ -80,9 +80,9 @@ option_list <- list(
     help = "Path to input SCE file in RDS format."
   ),
   make_option(
-    "--results_dir",
+    "--output_tsv_file",
```

Comment:
I tend to favor slightly shorter option names. So either `output_file` or `output_tsv`... But this is very minor.
```suggestion
    "--output_file",
```